src/main/java/core/basesyntax/service/impl/CsvReaderServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.CsvReaderService;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvReaderServiceImpl implements CsvReaderService {
+    private static final int CODE = 0;
+    private static final int FRUIT = 1;
+    private static final int QUANTITY = 2;
+
+    @Override
+    public List<FruitTransaction> readCsv(String pathFile) {
+        try (CSVReader reader =
+                     new CSVReaderBuilder(new FileReader(pathFile)).withSkipLines(1).build()) {
+            List<FruitTransaction> fruitTransactionList = new ArrayList<>();
+            String[] record;
+            while ((record = reader.readNext()) != null) {
```

Comment:
I do not see advantages of opencsv library 
you can use com.opencsv.bean.CsvToBeanBuilder;
see example
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/f5ae5851-056e-4fb1-a24e-fc058582fdd5)

Reply:
I can`t use enum. Can I will change enum to String?

Reply:
lets wait for Leonid hint in chat and fix this